modules/ROOT/pages/datagraph-qsg.adoc

```diff
@@ -2,13 +2,13 @@
 
 Use this quick start guide to begin using Anypoint DataGraph. With Anypoint DataGraph, you can query multiple APIs in a single request to get the specific information you need.
 
-Getting started with Anypoint DataGraph involves:
+Completing this quick start guide involves:
 
 . Downloading and publishing two APIs
-. Adding those APIs to Anypoint DataGraph and editing their schemas for addition to the unified schema, including how to use various . API schema editing tools like linking and merging
+. Adding those APIs to Anypoint DataGraph and editing their schemas for addition to the unified schema, including how to use various API schema editing tools like linking and merging
```

Comment:
Why not break this bullet item up so it's: 
* Adding those APIs...
* Using various API schema editing tools...

Reply:
hmm, good question, it's one point, tho, so let me suggest something